tst/testspecial/backtrace.g.out

```diff
@@ -109,6 +135,19 @@ for i in 1 do
 od; at *stdin*:24 called from
 <function "f">( <arguments> )
  called from read-eval loop at *errin*:1
+brk> WhereWithVars();
+for i in 1 do
+    return 1;
+od; at *stdin*:24
+with local state:
+-Arguments:
+-Locals:
+   i: <unassigned>
+
+
+ called from
```

Comment:
I think I'd prefer if there was only one or even no empty line above this, and instead some more indentation. But of course that's subjective and a matter of taste.